themes/SuiteP/include/EditView/EditView.tpl

```diff
@@ -170,7 +170,7 @@
                     </a>
 
                 </div>
-                <div class="panel-body {{$collapse}}" id="detailpanel_{{$panelCount}}">
+                <div class="panel-body {{$collapse} panelContainer}" id="detailpanel_{{$panelCount}}" data-id="{{$label_upper}}">
```

Comment:
panelContainer in between the smarty markers '{', '}'. Is it works properly?

Reply:
It actually does...surprisingly :D. Just fixed it now.